### PR TITLE
Update postman collection for v0.27.0

### PR DESCRIPTION
**Pull request by curquiza**
- Add typo-tolerance settings
- Add `cropMarker`, `higlightPreTag`, `highlightPostTag`

### CODE REVIEW

File: .vuepress/public/postman/meilisearch-collection.json [hunk 1]
```diff
@@ -1192,6 +1207,81 @@
...
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"minWordLengthForTypo\": {\n        \"oneTypo\": 2,\n        \"twoTypos\": 11\n    },\n    \"disableOnWords\": [\"Prince\"],\n    \"disableOnAttributes\": [\"genre\"]\n\n}"
```
Comment (maryamsulemani97) [inline .vuepress/public/postman/meilisearch-collection.json:1249]:
```suggestion
									"raw": "{\n    \"minWordSizeForTypos\": {\n        \"oneTypo\": 2,\n        \"twoTypos\": 11\n    },\n    \"disableOnWords\": [\"Prince\"],\n    \"disableOnAttributes\": [\"genre\"]\n\n}"
```

Shouldn't this be `minWordSizeForTypos`?

Comment (curquiza) [reply .vuepress/public/postman/meilisearch-collection.json:1249]: Sorry, I did it using v0.27.orc1, I should have waited for rc2 before posting this PR. Changing this!

File: .vuepress/public/postman/meilisearch-collection.json [hunk 2]
```diff
@@ -286,6 +291,16 @@
 									"key": "sort",
 									"value": "price",
 									"disabled": true
+								},
+								{
+									"key": "highlightPreTag",
+									"value": "<mark>",
+									"disabled": true
```
Comment (maryamsulemani97) [inline .vuepress/public/postman/meilisearch-collection.json:298]: Idk if it's supposed to be different here, but shouldn't this be `<em>`?

File: .vuepress/public/postman/meilisearch-collection.json [hunk 3]
```diff
@@ -286,6 +291,16 @@
...
+								},
+								{
+									"key": "highlightPostTag",
+									"value": "</mark>",
```
Comment (maryamsulemani97) [inline .vuepress/public/postman/meilisearch-collection.json:302]: And this `</em>`?

**Review by maryamsulemani97 — APPROVED** [review]
🎈

**Pull request merged**